docs/calculations.mdx

```diff
@@ -437,42 +437,55 @@ creative_consumer_device_embodied_emissions_gco2pm =
 ### Media Delivery - Data Transfer
 
 ```
-session_seconds_per_imp =
-      property.average_seconds_per_session_ex_ads x
-      property.ad_funded_percentage /
-      property.average_imps_per_session
-
 media_data_transfer_emissions_gco2pm =
       media_data_transfer_usage_emissions_gco2pm +
       media_data_transfer_embodied_emissions_gco2pm
+```
+
+Determing the session seconds for a single impressions
 
 ```
+session_seconds_per_imp =
+      property.average_seconds_per_session_ex_ads x
+      property.ad_funded_percentage /
+      property.average_imps_per_session
+```
 
 For all channels other than CTV/BVOD, use the conventional model:
 
 ```
+media_data_transfer_kb_per_imp =
+      property.average_data_kb_per_session_seconds_ex_ads x
+      session_seconds_per_imp
+
 media_data_transfer_usage_emissions_gco2pm =
-      property.average_data_kb_per_session_ex_ads x
+      media_data_transfer_kb x
       usage_kwh_per_gb / 1000000 x
       lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
 
 media_data_transfer_embodied_emissions_gco2pm =
-      property.average_data_kb_per_session_ex_ads x
+      media_data_transfer_kb x
       embodied_emissions_gco2e_per_kb
```

Comment:
Should be a data transfer bytes/kb number here so broke out calculating `media_data_transfer_kb_per_imp` above to then use in these calculations